app/controllers/submit.js

```diff
@@ -525,11 +541,12 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                     action: 'click',
                     label: 'Preprints - Submit - Save and Continue, New Component, Copy File'
                 });
-            node.addChild(this.get('nodeTitle'))
+            node.addChild(fixSpecialChar(this.get('nodeTitle')))
```

Comment:
Is this really necessary? Bleach lets existing escape sequences pass untouched. There's a chance that setting the new node title to the old title will give you back the desired result regardless...